packages/ui-extensions/src/surfaces/point-of-sale/components/TimeField/TimeField.ts

```diff
@@ -14,6 +14,10 @@ export interface TimeFieldProps
     | 'action'
     | 'helpText'
   > {
+  /**
+   * (Android only) Whether the clock displays in 24 hour time instead of 12 hour time. 
```

Comment:
```suggestion
   * (Android only) Whether the clock displays in 24 hour time instead of 12 hour time. 
```

I think this is what the linter is complaining about
![image](https://github.com/Shopify/ui-extensions/assets/29490857/d3a28d5a-aa03-4092-b9f3-576ad6430acf)